playbooks/configure-apt-sources.yml

```diff
@@ -115,7 +128,14 @@
         - apt_artifact_found | bool
         - apt_artifact_enabled | bool
 
-    - name: Replace the apt sources file with our content
+    - name: Backup the original sources file
```

Comment:
I don't think this task should be part of this PR. The restore actions we're proposing in #2720 are aiming to address this issue. From the POV of this PR it make sense to simply make the modifications as it is today when enabled, unless we want to merge these two PRs.

Reply:
That PR would be a good evolution over and above what's being done here because that takes the flipping further. This is only addressing the apt package flip between strict and loose mode, it does not handle removing the setting of artifacts all together.